ideascube/serveradmin/tests/test_models.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime, timezone
+
+import freezegun
+import pytest
+
+
+@pytest.mark.parametrize(
+    'value, type_method',
+    [
+        (True, 'as_boolean'),
+        (42, 'as_integer'),
+        (None, None),
+        ('A string', 'as_string'),
+    ],
+    ids=[
+        'boolean',
+        'int',
+        'none',
+        'string',
+    ])
+@pytest.mark.usefixtures('db')
+def test_settings(value, type_method, user):
+    from ideascube.serveradmin.models import Setting
+
+    fakenow = datetime.now(tz=timezone.utc)
+    assert Setting.objects.count() == 0
+
+    with freezegun.freeze_time(fakenow):
```

Comment:
I think you can simply use `Setting.objects.update(date=mydate)` to avoid this new dep.